bench.list

```diff
@@ -0,0 +1,23 @@
+[benchmarks]
+    [./simple_diffusion_refine3]
+        binary = test/moose_test-opt
```

Comment:
I'm OK with this right now since it's fairly stable but I'd prefer that we just pick it up. Let's not change this if we go with a version of this current script because you'll just be recoding logic that already exists in the TestHarness. If we switch systems, this will become automatic.